variables.tf

```diff
@@ -23,6 +23,12 @@ variable "zone" {
   description = "The zone that instances should be created in"
 }
 
+variable "machine_image" {
```

Comment:
I think that we should tweak this a bit, as now it looks like you could pass any machine image reference, but we're checking agains the `cos-cloud` project under the hood. It could be a bit misleading to the user.

Reply:
I'll take a stab at testing this now